### ISSUE TITLE
Verification of Installation, Back- Up and Clean Up Steps necessary for Successful install of v1.8

### ISSUE THREAD

**Issue by cflottorp** [NASA-Tournament-Lab/NTL-ISS-Food-Intake-Tracker#196]
Need to confirm the order for the installation, back up of the existing pl_fit-db and clean up of database using the various scripts. Due to the various scripts that have been delivered, the order of the recommended deployment needs to be verified. Please confirm the following order or update as needed for a "clean", rebuild of the ISS FIT  for version 1.8. 
1.	Run this database migration script     https://github.com/NASA-Tournament-Lab/NTL-ISS-Food-Intake-Tracker/blob/master/Backend Services/Migration.md  
2.	Run the CleanUpISSFITDatabase script
3.	Run the 0100_fit_v1 script
4.	Run the 0100_fit_v2 script
5.	Using the Admin Tool, upload a Food CSV file and keep the checkbox "mark all food as removed" checked

**Comment by rsial2** [comment]
I would make number 1 "back up database".

For the others, I'll ask @coderReview to confirm.

**Comment by coderReview** [comment]
@cflottorp 
if the server is already running the FIT application step 3 is not necessary. The v1 script is to setup the first version of the FIT application which is running already.

Since step 4 will delete the database, the migration step (1) won't do any good. If you want I can remove the database removal from the v2 script. 

I don't know what is the script in step 2.

In my opinion, if you want to keep the data for the next version, you will need to:

1- Run the database migration
2- Run the 0100_fit_v2 script
3- Using the Admin Tool, upload a Food CSV file and keep the checkbox "mark all food as removed" checked

Otherwise you will need to:

1- Run the 0100_fit_v2 script
2- Using the Admin Tool, upload a Food CSV file

**Comment by cflottorp** [comment]
The script in step 2 was to clean up the ISS FIT database to remove duplication in the database.  With the first recommendation to run the migration, 0100_fit_v2 script , will that include cleaning up the database to remove the duplicates?

**Comment by rsial2** [comment]
@coderReview we need to push to resolve this as soon as possible - please confirm.

**Comment by coderReview** [comment]
@cflottorp can you confirm if you want to keep the current data? From the other issue I understood you would like to start fresh.

**Comment by cflottorp** [comment]
With the V1 and v2 scripts there is no need to run the clean up script in step 2 that was received previously? This was a script to remove duplicates in the database.  https://drive.google.com/file/d/0B3nmbMgH4_oxWVV6bFdVZkRGWmM/view

**Comment by cflottorp** [comment]
We  will be doing a fresh re-load, so we should only need to be running the  0100_fit_v2 script.  Then we will use Admin Tool to upload a Food CSV file and keep the checkbox "mark all food as removed" checked and also a User Data CSV file for user log in.  Correct?

**Comment by coderReview** [comment]
Correct. You may just want to make a backup of the existing data for historical reason. I've sent the command on how to do that some time ago. Do you need me to send again? @rsial2 since the old data won't be kept the script is good to go for testing.

**Comment by rsial2** [comment]
We need @szwart to confirm if the old data is needed or not.

**Comment by cflottorp** [comment]
Yes, we use the "pg_dump -h $' IP address' -p xxxxx -U pl_fit_db -W -Fc -f pl_fit.bak pl_fit"

**Comment by szwart** [comment]
We are good with a total wipe and reload.  We get reports down every week so we have all of the historical data we need.
